src/_cffi_src/openssl/pem.py

```diff
@@ -82,7 +82,11 @@
                                unsigned char *, int, pem_password_cb *,
                                void *);
 int PEM_write_bio_DHparams(BIO *, DH *);
+int PEM_write_bio_DHxparams(BIO *, DH *);
 """
 
 CUSTOMIZATIONS = """
+#ifndef EVP_PKEY_DHX
```

Comment:
EVP_PKEY_DHX gets defined if it doesn't exist in `evp.py`. This check is in `dh.py`, which gets loaded first (mostly because we decided to list the modules mostly alphabetically), but I'm not super comfortable with the bindings being order dependent in this fashion. Maybe we need `#if !defined(EVP_PKEY_DHX) && EVP_PKEY_DHX != -1` as a guard?